packages/itwinui-css/src/utils/icon/icon.scss

```diff
@@ -18,14 +18,31 @@ $iui-icon-sizes: (
   3xl: var(--iui-size-2xl)
 );
 
-/// @arg $size Can be one of: s, m, l, xl, 2xl, 3xl
-@mixin iui-icon-style($size: 'm') {
+/// @arg $size Can be one of: s, m, l, xl, 2xl
+@mixin iui-icon-style($size) {
```

Comment:
This mixin can be used inside other components for easily getting the size, flex, etc.